src/transformers/models/convnext/modeling_tf_convnext.py

```diff
@@ -383,7 +383,8 @@ def serving(self, inputs):
             inputs (`Dict[str, tf.Tensor]`):
                 The input of the saved model as a dictionary of tensors.
         """
-        return self.call(inputs)
```

Comment:
Adding in `self.serving(output)` return to: 
1. keep in line with the rest of the models call pattern and return types e.g. not including loss (I don't know why this is done) 
2. allow for testing (see PR desc.)